tests/conftest.py

```diff
@@ -95,14 +95,52 @@ def build(
 @dataclass
 class AWS:
     access_key: str
-    secret_key: str
-
-
-@pytest.fixture(name='real_aws')
-def _fix_real_aws():
-    access_key = os.getenv('TEST_AWS_ACCESS_KEY')
-    secret_key = os.getenv('TEST_AWS_SECRET_KEY')
-    if access_key and secret_key:
-        return AWS(access_key, secret_key)
+    secret_key: str = field(repr=False)
+    session_token: str = field(repr=False, default='')
+
+
+@pytest.fixture(
+    name='real_aws',
+    params=(
+        ('TEST_AWS_ACCESS_KEY', 'TEST_AWS_SECRET_KEY', ''),
+        ('TEST_AWS_SESSION_ACCESS_KEY', 'TEST_AWS_SESSION_SECRET_KEY', 'TEST_AWS_SESSION_TOKEN'),
+    ),
+)
+def _fix_real_aws(request):
+    access_key_variable_name, secret_key_variable_name, session_token_variable_name = request.param
+    access_key = os.getenv(access_key_variable_name)
+    secret_key = os.getenv(secret_key_variable_name)
+    session_token = os.getenv(session_token_variable_name) if session_token_variable_name else ''
+    if (access_key and secret_key) and session_token is not None:
+        return AWS(access_key, secret_key, session_token)
     else:
         pytest.skip('requires TEST_AWS_ACCESS_KEY & TEST_AWS_SECRET_KEY env var')
+
+
+@pytest.fixture(name='real_aws_s3_bucket_name')
+def _fix_real_aws_s3_bucket_name():
+    """Configure AWS S3 bucket name for testing.
+
+    A configurable S3 bucket name enables contributors to run tests on their forks
+    and in their AWS accounts, by setting `${{ secrets.TEST_AWS_S3_BUCKET_NAME }}`
+    for GitHub Actions. Defaults to `aioaws-testing`.
+
+    The function is written with an if expression instead of using `os.getenv` with a
+    default value (`os.getenv('TEST_AWS_S3_BUCKET_NAME', 'aioaws-testing')`) because
+    of how the environment variable value is passed in from GitHub Actions. If
+    `${{ secrets.TEST_AWS_S3_BUCKET_NAME }}` is not set, `TEST_AWS_S3_BUCKET_NAME`
+    will be an empty string, and `os.getenv('TEST_AWS_S3_BUCKET_NAME', 'aioaws-testing')`
+    will return an empty string.
+    """
+    return value if (value := os.getenv('TEST_AWS_S3_BUCKET_NAME')) else 'aioaws-testing'
+
+
+@pytest.fixture(name='real_aws_ses_address')
+def _fix_real_aws_ses_address():
+    """Configure AWS SES email address for testing.
+
+    As with the S3 bucket name, enabling configuration of the email address allows
+    contributors to run tests on their forks and in their AWS accounts, by setting
+    `${{ secrets.TEST_AWS_SES_ADDRESS }}` on GitHub. Defaults to `testing@scolvin.com`.
+    """
+    return value if (value := os.getenv('TEST_AWS_SES_ADDRESS')) else 'testing@scolvin.com'
```

Comment:
```suggestion
    return os.getenv('TEST_AWS_SES_ADDRESS', 'testing@scolvin.com')
```

Reply:
Same comment as for `TEST_AWS_S3_BUCKET_NAME`: https://github.com/samuelcolvin/aioaws/pull/23#discussion_r763470525